libraries/AP_Scripting/lua/src/lobject.c

```diff
@@ -63,6 +63,13 @@ int luaO_fb2int (int x) {
 ** Computes ceil(log2(x))
 */
 int luaO_ceillog2 (unsigned int x) {
+  // never called with 0
+#if defined(__GNUC__) || defined(__clang__)
+  const int bitwidth = CHAR_BIT*sizeof(x);
+  x--;
+  const int clz = x ? __builtin_clz(x) : bitwidth; // x == 0 is undefined
+  return bitwidth-clz;
+#else
```

Comment:
its not clear by inspection that these are the same. I wonder if we could run both in SITL and panic if they don't match. 

We also typically have been putting `#if defined(ARDUPILOT_BUILD)` round things we change in the lua source just so we can find it again. 

I'm not sure this is worth 300 bytes, on the scale of how much scripting costs its a small fraction.

Reply:
It is going to be faster too. I did a bunch of tests to confirm that it’s the same but I can see the apprehension.

I’m probably going to bring this back as a separate PR.

Reply:
This stands well on its own, dropping for now.